rsearch/conductor.go

```diff
@@ -0,0 +1,43 @@
+package rsearch
+
+func manageResources(ns Event, terminators map[string]chan Done, config Config, out chan Event) {
+	uid := ns.Object.Metadata.Uid
+	if ns.Type == "ADDED" {
+		done := make(chan Done)
+		terminators[uid] = done
+		ns.Object.Produce(out, terminators[uid], config)
+	} else if ns.Type == "DELETED" {
+		close(terminators[uid])
+		delete(terminators, uid)
+	} else if ns.Type == "_CRASH" {
```

Comment:
Defining constants, rather than using string literals, is definitely an improvement.